.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on: push
+
+jobs:
+  specs:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby-version:
+          - "2.7"
```

Comment:
This looks like overkill for testing just one Ruby version, but I suspect you want to test a few more in a subsequent PR?

Reply:
I do, yep!